tests/pyfunc/test_scoring_server.py

```diff
@@ -153,6 +153,32 @@ def test_scoring_server_successfully_evaluates_correct_dataframes_with_pandas_sp
     assert response.status_code == 200
 
 
+@pytest.mark.large
+def test_scoring_server_successfully_evaluates_correct_records_to_numpy(
```

Comment:
nit: test_scoring_server_successfully_evaluates_correct_dataframes_with_pandas_records_orientation_parsed_to_numpy_array

Reply:
The name is too long to fit in one line :)